test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -568,9 +591,11 @@ TEST(LiteInterpreterTest, HierarchyModuleInfo) {
   // There are 3 module information strings here.
   // "top(C).forward": for the add operator in top.
   // "top(C).B0(B).forward": for the add operator in B0.
-  // "top(C).B0(B).A0(A).forward": for the add operator in A0.
+  // "top(C).B0(B).forward.A0(A).forward": for the add operator in A0.
   std::unordered_set<std::string> expected_result(
-      {"top(C).forward", "top(C).B0(B).forward", "top(C).B0(B).A0(A).forward"});
+      {"top(C).forward",
+       "top(C).B0(B).forward",
+       "top(C).B0(B).forward.A0(A).forward"});
```

Comment:
This should also probably not exist.

Reply:
Which part do you refer to? I think the expected result for this test case is `"top(C).B0(B).forward.A0(A).forward"`, because B0 is a submodule of C, and A0 is a submodule of B.